src/main/java/core/basesyntax/OperationStrategy.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import core.basesyntax.impl.BalanceOperation;
+import core.basesyntax.impl.PurchaseOperation;
+import core.basesyntax.impl.ReturnOperation;
+import core.basesyntax.impl.SupplyOperation;
+
+public class OperationStrategy {
+    public OperationHandler getOperation(String operationType) {

```

Comment:
Let's use variable `operationHandlers` here to return an appropriate handler for transaction
```suggestion
    public OperationHandler getOperation(FruitTransaction fruitTransaction) {
```
FruitTransaction is a simple class that stores information about what we want to do with our storage. It has 3 fields:
```
    private Operation operation; // inner enum
    private String fruit;
    private int quantity;
```

```
enum Operation {
        BALANCE("b"),
        SUPPLY("s"),
        PURCHASE("p"),
        RETURN("r");
 }
```
 This enum should also have a method that returns the appropriate value based on the "code" we pass in ("b", "p", "r", "s")